package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "restful.js",
   "version": "0.9.2",
-  "repository": "https://github.com/marmelab/restful.js",
+  "repository": {
+    "type": "git",
```

Comment:
Not mandatory, just compliance. Here is schema definition for package.json http://json.schemastore.org/package